### ISSUE TITLE
Only 2nd test is passed

### ISSUE THREAD

**Issue by rmmadcas** [wesbarnett/libgmxfort#3]
Hello!
The first thing is to thank you for your contribution. I have been able to install your libraries and use them in my Linux but when I try to install it in the university cluster only the second test passes. The others fail. Thanks again

**Comment by wesbarnett** [comment]
Hi, can you give the exact output?

**Comment by rmmadcas** [comment]
Running tests...
Test project /home/ucecmad/xtc_way/libgmxfort/build
    Start 1: Read_xtcfile
1/5 Test #1: Read_xtcfile .....................***Failed    0.12 sec
    Start 2: Read_xtcfile_and_ndxfile
2/5 Test #2: Read_xtcfile_and_ndxfile .........   Passed    0.24 sec
    Start 3: Read_ndx_grp
3/5 Test #3: Read_ndx_grp .....................***Failed    0.24 sec
    Start 4: Read_next
4/5 Test #4: Read_next ........................***Failed    0.24 sec
    Start 5: Utils
5/5 Test #5: Utils ............................***Failed    0.25  sec
20% tests passed, 4 tests failed out of 5
Total
 Test time (real) =   1.11 sec

The following tests FAILED:
	  1 - Read_xtcfile (Failed)
	  3 - Read_ndx_grp (Failed)
	  4 - Read_next (Failed)
	  5 - Utils (Failed)
Errors while running CTest

make: *** [test] Error 8

**Comment by wesbarnett** [comment]
Do the following:
```
cd /home/ucecmad/xtc_way/libgmxfort/src/tests
../../build/src/tests/xtcfile
../../build/src/tests/ndxfile
../../build/src/tests/ndxfile_grp
../../build/src/tests/read_next
../../build/src/tests/utils
```
and then post the output

**Comment by rmmadcas** [comment]
[ucecmad@login02 tests]$ cd /home/ucecmad/xtc_way/libgmxfort/src/tests
[ucecmad@login02 tests]$ ../../build/src/tests/xtcfile
Opened test.xtc for reading.
16500 atoms present in system.
101 frames present in trajectory file.
Frames saved: 101
Closed xtc file.
LIBDCDFORT ERROR:  is not in index file. The groups available are:
1
[ucecmad@login02 tests]$ ../../build/src/tests/ndxfile
Opened test.xtc for reading.
16500 atoms present in system.
101 frames present in trajectory file.
Frames saved: 101
Closed xtc file.
LIBDCDFORT WARNING: No atoms found in index group ''.
This warning will not appear again for any other index groups.
TEST  1: PASSED
TEST  2: PASSED
TEST  3: PASSED
TEST  4: PASSED
TEST  5: PASSED
TEST  6: PASSED
TEST  7: PASSED
 
Passed 7 out of 7 tests
 
[ucecmad@login02 tests]$ ../../build/src/tests/ndxfile_grp
Opened test.xtc for reading.
16500 atoms present in system.
101 frames present in trajectory file.
Frames saved: 101
Closed xtc file.
LIBDCDFORT ERROR:  is not in index file. The groups available are:
    System (16500)
     Water (16500)
       SOL (16500)
      TEST (1)
        OW (4125)
     TEST2 (1)
1
[ucecmad@login02 tests]$ ../../build/src/tests/read_next
Opened test.xtc for reading.
16500 atoms present in system.
101 frames present in trajectory file.
Frames saved: 100
LIBDCDFORT ERROR:  is not in index file. The groups available are:
    System (16500)
     Water (16500)
       SOL (16500)
      TEST (1)
        OW (4125)
     TEST2 (1)
1

**Comment by wesbarnett** [comment]
What compiler did you use? gfortran? Also what version was it?

**Comment by wesbarnett** [comment]
This is odd. I'm not sure how it's trying to read in the index file in the first test.

**Comment by rmmadcas** [comment]
GNU Fortran (GCC) 4.9.2

**Comment by wesbarnett** [comment]
The only guidance I can probably give is to try to use a more recent version of GNU Fortran.

Also, double check the contents of `src/tests/test.ndx` to make sure they match what is in this repo [here](https://raw.githubusercontent.com/wesbarnett/libgmxfort/master/src/tests/test.ndx).

On the surface it looks like for some reason there is an additional group in your version of the test index file named as an empty string with 1 atom in it. But, it could also be that I might be using some feature not available in 4.9.2, but I can't nail down what that is.
